README.md

```diff
@@ -256,7 +256,9 @@ $ LD_PRELOAD=/usr/lib/libumf_proxy.so myprogram
 
 The memory used by the proxy memory allocator is mmap'ed:
 1) with the `MAP_PRIVATE` flag by default or
-2) with the `MAP_SHARED` flag only if the `UMF_PROXY` environment variable contains the `page.disposition=shared` string.
+2) with the `MAP_SHARED` flag if the `UMF_PROXY` environment variable contains one of two following strings: `page.disposition=shared-shm` or `page.disposition=shared-fd`. These two options differ in a mechanism used during IPC:
+   - `page.disposition=shared-shm` - then the named shared memory is used during IPC. An SHM name is generated using the `umf_proxy_lib_shm_pid_$PID` pattern, where `$PID` is the PID of the process. It creates the `/dev/shm/umf_proxy_lib_shm_pid_$PID` file.
```

Comment:
"then" -> "in this case" or " - IPC use the named shared... "

Reply:
Done